lib/modulesync.rb

```diff
@@ -80,7 +82,7 @@ def self.run(args)
         files_to_manage -= files_to_delete
         if options[:noop]
           Git.update_noop(puppet_module, options)
-        else
+        elsif not options[:offline]
```

Comment:
So, if we do that, we're changing the way --noop actually works.  Since now, we get output with both --noop or --offline.

This way, you have to pass both --offline --noop if you want to see the diff.  Other wise, --offline does nothing related to git.